src/Microsoft.OData.Core/UriParser/Binders/SelectPathSegmentTokenBinder.cs

```diff
@@ -185,7 +185,7 @@ internal static bool TryBindAsOperation(PathSegmentToken pathToken, IEdmModel mo
             // If more than one overload matches, try to select based on optional parameters
             if (possibleFunctions.Count() > 1 && parameterNames.Count > 0)

```

Comment:
>Count [](start = 34, length = 5)

Like this too.

Reply:
Reverted the CountGreaterThan change. Experimental data showed that .Count() outperforms CountGreaterThan for smaller values. 

---
In reply to: [354091432](https://github.com/OData/odata.net/pull/1602#discussion_r354091432) [](ancestors = 354091432)